projects/lib/src/oauth-service.ts

```diff
@@ -1798,10 +1750,16 @@ export class OAuthService extends AuthConfig implements OnDestroy {
             }
           },
           err => {
+            // TODO: decide which version is correct
+            // --- password flow
+            this.logger.error('Error performing password flow', err);
+            this.eventsSubject.next(new OAuthErrorEvent('token_error', err));
+            // --- code flow
             console.error('Error getting token', err);
             this.eventsSubject.next(
               new OAuthErrorEvent('token_refresh_error', err)
             );
+            // ---
```

Comment:
During the merge of the processing functions I encountered this conflicting area with different events.
I'm not sure which option is the right one here, maybe someone can help.